package.json

```diff
@@ -50,23 +55,30 @@
     "gulp-sequence": "1.0.0",
     "gulp-sizereport": "1.2.0",
     "gulp-sourcemaps": "^2.6.0",
+    "gulp-stylelint": "^6.0.0",
     "gulp-svgstore": "6.1.0",
-    "gulp-watch": "4.3.11",
     "lodash": "^4.17.4",
     "merge-stream": "^1.0.1",
     "minimist": "^1.2.0",
     "node-fs-extra": "^0.8.2",
     "node-sass-glob-importer": "^5.0.0-alpha.5",
     "open": "0.0.5",
     "plugin-error": "^0.1.2",
+    "prettier": "^1.10.2",
     "pretty-hrtime": "1.0.3",
     "require-dir": "0.3.2",
-    "webpack": "^3.4.1",
+    "stylelint": "^8.4.0",
+    "stylelint-order": "^0.8.0",
+    "webpack": "3.10.0",
     "webpack-dev-middleware": "^1.12.0",
     "webpack-hot-middleware": "^2.18.2"
   },
   "devDependencies": {
     "chai": "^3.5.0",
     "mocha": "3.2.0"
+  },
+  "prettier": {
+    "semi": false,
+    "singleQuote": true
```

Comment:
prettier runs as a text editor plugin, so this will only work if users has this installed